<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Publicaciones</class>
 <widget class="QMainWindow" name="Publicaciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(216, 229, 158);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>670</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>REGRESAR</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>421</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sitka&quot;;</string>
     </property>
     <property name="title">
      <string>FECHA</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>APLICAR</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbFecha">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 5px;
font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
text-align: center;
</string>
      </property>
      <item>
       <property name="text">
        <string>TODO</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sitka&quot;;</string>
     </property>
     <property name="title">
      <string>RECORRIDO LIMITADO</string>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>APLICAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>ORDEN:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>CANTIDAD:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCantidad">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbOrden">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">text-align: center;
font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 5px;
</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PreOrden</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PosOrden</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>InOrden</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>781</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnComentar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>520</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string>COMENTAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVercoment">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>520</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string>VER COMENTARIOS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVerArbol">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>520</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string>VER ARBOL COMENTARIOS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCrearPub">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string>CREAR UNA PUBLICACION</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plnText">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>711</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUsuario">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>321</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 12pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>TEXTO PUBLICACION: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 12pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>USUARIO: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFecha">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>321</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFecha">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 12pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>FECHA: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblImagen">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>391</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 20pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSiguiente">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>410</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string>SIGUIENTE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAnterior">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>410</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string>ANTERIOR</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
